sdk/servicelinker/Azure.ResourceManager.ServiceLinker/src/autorest.md

```diff
@@ -8,18 +8,40 @@ azure-arm: true
 csharp: true
 library-name: ServiceLinker
 namespace: Azure.ResourceManager.ServiceLinker
-require: https://github.com/Azure/azure-rest-api-specs/blob/eb2b882ef0a4aa5956ca38cfa566fc4d7cfb3fb0/specification/servicelinker/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/42ca0236ef14093f5aff0694efa34d5594e814a0/specification/servicelinker/resource-manager/readme.md
 tag: package-2022-05-01
 output-folder: $(this-folder)/Generated
 clear-output-folder: true
 skip-csproj: true
 modelerfour:
   flatten-payloads: false
 
+rename-mapping:
+  TargetServiceBase: TargetServiceBaseInfo
+  AzureResource: AzureResourceInfo
+  ConfluentBootstrapServer: ConfluentBootstrapServerInfo
+  ConfluentSchemaRegistry: ConfluentSchemaRegistryInfo
+  AzureResourcePropertiesBase: AzureResourceBaseProperties
+  SecretInfoBase: SecretBaseInfo
+  ValueSecretInfo: RawValueSecretInfo
+  AuthInfoBase: AuthBaseInfo
+  AuthType: LinkerAuthType
+  SecretType: LinkerSecretType
+  SecretStore: LinkerSecretStore
+  ClientType: LinkerClientType
+  ValidationResultItem: ValidationResultItemData
```

Comment:
Why did we use the `Data` suffix here? It looks like that this model doesn't inherits from any `Data` type in ResourcceManager.

Reply:
In the `KeyVault` SDK Ted considered to avoid use `Item` suffix, so append `Data`, so here do the same rename